### PR TITLE
feat: add FLUX.1

### LINKED ISSUE #55: Support FLUX.1
**Is your feature request related to a problem? Please describe.**

I want to use FLUX.1 via an API. 

**Describe the solution you'd like**

I have an image that already contains the models needed to run FLUX.1. 

**Describe alternatives you've considered**

Forking this repo and creating the image myself.

### PR DESCRIPTION
**Pull request by TimPietrusky**
## Motivation

* Add images for FLUX.1-dev and FLUX.1-schnell

## Issues closed

* Fixes #55

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -57,6 +57,11 @@ RUN if [ "$MODEL_TYPE" = "sdxl" ]; then \
       wget -O models/vae/sdxl-vae-fp16-fix.safetensors https://huggingface.co/madebyollin/sdxl-vae-fp16-fix/resolve/main/sdxl_vae.safetensors; \
     elif [ "$MODEL_TYPE" = "sd3" ]; then \
       wget --header="Authorization: Bearer ${HUGGINGFACE_ACCESS_TOKEN}" -O models/checkpoints/sd3_medium_incl_clips_t5xxlfp8.safetensors https://huggingface.co/stabilityai/stable-diffusion-3-medium/resolve/main/sd3_medium_incl_clips_t5xxlfp8.safetensors; \
+    elif [ "$MODEL_TYPE" = "flux1-schnell" ]; then \
+      wget -O models/unet/flux1-schnell.safetensors https://huggingface.co/black-forest-labs/FLUX.1-schnell/resolve/main/flux1-schnell.safetensors && \
```
Comment (hoblin) [inline Dockerfile:61]: You'll probably want to add `  unet: models/unet/` to the src/extra_model_paths.yaml

Comment (TimPietrusky) [reply Dockerfile:61]: @hoblin that makes sense, will add this. 

My overall plan is to also have an image with flux1-dev, I just ran out of time testing the basic version with just FLUX.1-schnell. But adding `unet` into the mapping is still valuable anyway.

Comment (hoblin) [reply Dockerfile:61]: BTW if you have problems running `-basic` container you might try this fix https://github.com/blib-la/runpod-worker-comfy/issues/54#issuecomment-2282829872

**Comment by TimPietrusky** [comment]
`FLUX.1-schnell` is working as desired, now let's make sure `FLUX.1-dev` is also working.

**Pull request merged**